components/grid/export/pdf.md

```diff
@@ -103,6 +107,26 @@ Optionally, you can also set the `GridPdfExport` tag settings under the `GridExp
 }
 ````
 
+## How the Export Works
+
+The PDF export has the following specifics:
+
+* The UI for Blazor PDF export will export the Grid to a table and will not export the full HTML. This behavior is different from when exporting in Kendo UI for jQuery. [To export to PDF as HTML, you can use a custom approach](#custom-export).
+
+## Requirements
+
+The PDF export has the following requirements:
+
+* PDF export requires pixel widths for all columns. Widths in other units such as `%` or `em` cannot be translated correctly and the respective columns will collapse in the exported PDF file. The column widths for the PDF export can differ from the ones in the Grid configuration for the web browser. To set column widths for the PDF file only, use the `Width` property of the [`OnBeforeExportEventArgs.Columns`](slug:grid-export-events#for-pdf-export) members.
+* Provide appropriate `PaperSize` and `PageOrientation` properties. For example, if you want to render 20 columns (100px each) in an A4 sheet, then this will yield unexpected results. The column dimensions in a PDF file are fixed, thus they cannot be resized as in Excel, which requires the developer to ensure proper export dimensions.
```

Comment:
I would mention / link this briefly in the basics section, in order to ensure that readers who experiment with PDF export for the first time and code while reading, will not hit something that they can perceive as a bug.